engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,22 +236,48 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float getJumpSpeed(float amount){
+        double d = (double)amount;
+        return (float) pow(d,0.6) * 9.36f ;
```

Comment:
The default `jumpSpeed` seems to be 10 (and that's the maximum at the same time o.O ). thus, I would assume that setting the player height to the default should result in the default jump speed, but substituting `amount` by 1.6 yiels a value ~12.4 (≠10).

This is how the jump speed is adjusted according to this function, for `amount` in the interval from 0 to 5:
![image](https://user-images.githubusercontent.com/1448874/78274288-470a3800-7510-11ea-881c-092fc9242763.png)
see [Wolfram Alpha](https://www.wolframalpha.com/input/?i=pow%28x%2C+0.6%29+*+9.36%2C+x+in+%5B0%2C5%5D)


see [CharacterMovementComponent.java#L53-L54](https://github.com/godfather2/Terasology/blob/heightExtras/engine/src/main/java/org/terasology/logic/characters/CharacterMovementComponent.java#L53-L54)